### PR TITLE
Adding frame playback

### PR DESCRIPTION
**Pull request by toledoal**
Hello 😄 !

In this PR I continue integrating the design redlines. It also adds the dragging behavior of the play-head and bug fixes for adding animations.

### CODE REVIEW

File: inspector/src/components/actionTabs/tabs/propertyGrids/animations/playhead.tsx [hunk 1]
```diff
@@ -1,27 +1,95 @@
...
+    e.preventDefault();
+    this._direction = 0;
+    this._active = false;
+  }
 
+  calculateMove() {
+    return `calc(${this.props.frame * this.props.offset}px - 13px)`;
+  }
+
+  moveFrameTo(e: any) {
```
Comment (deltakosh) [inline inspector/src/components/actionTabs/tabs/propertyGrids/animations/playhead.tsx:54]: why any? we should avoid every any

Comment (toledoal) [reply inspector/src/components/actionTabs/tabs/propertyGrids/animations/playhead.tsx:54]: good catch,. Thanks! I missed that

**Comment by deltakosh** [comment]
Seems something is wrong in the tests

**Comment by toledoal** [comment]
let me check it here as well,

**Comment by toledoal** [comment]
Yeah I missed to remove one unused import

**Pull request merged**